The development-environment doesn't work on windows

When trying to test the module on a windows platform, weird errors appear as seen in AppVeyor:
https://ci.appveyor.com/project/benmosher/eslint-plugin-import/builds/33096831/job/gg7jvh5ehfb4s9ag

This also might be because the dependencies weren't updated for a long time and don't support `node 14.x`.

I'm talking 'bout these deprecation-warnings:
```
npm WARN deprecated typescript-eslint-parser@22.0.0: Deprecated: Use @typescript-eslint/parser instead
npm WARN deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
npm WARN deprecated json3@3.3.2: Please use the native JSON object instead of JSON 3
npm WARN deprecated typescript-estree@18.0.0: This package was moved to @typescript-eslint/typescript-estree, please install the latest version from there instead
npm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead
npm WARN deprecated samsam@1.3.0: This package has been deprecated in favour of @sinonjs/samsam
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated text-encoding@0.6.4: no longer maintained
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
```

Most importantly `chokidar@1.7.0` and `fsevents@1.2.13` as they have something like "will break on node 14+" in their messages.

Anyways - these are the issues I'm facing:
* When using windows w/ node 14 I get this error when running `npm run test`:  
 `Cannot find module 'E:\Users\Manuel\Documents\Projekte\Node-Modules\eslint-plugin-import\node'`
* When using `node:14` the container as well as the docker service get stuck

Also the `copy-metafiles`-script doesn't work on windows.

As far as I can see, the error `Error: Cannot find module 'C:\projects\eslint-plugin-import\node'` which appears on every windows-based appveyor test is related to babel.

I'm quite sure because `cjs` is the name of babel's `CommonJS` module loader. Additionally the error disappeared once I updated babel to the latest version.

Although I did not test whether it works if babel is updated to any other version.

Sadly build-stages aren't possible in AppVeyor - its functions seem to be very limited.
As far as I can see, script-stacks are executed in the following order:
* `init`
* _Clone git into a clone folder_
* `install`
* `before_build`
* `build_script`
* `after_build`
* `before_test`
* **`test_script`** (this is the most important stack in our case as currently this stack contains all the mocha-stuff atm)
* `after_test`
* `before_deploy`
* `after_deploy`
* `on_success`
* `on_failure`
* `on_finish`
As seen on https://www.appveyor.com/docs/build-configuration/

You can create multiple jobs using the `configuration` array (so did I in order to create tests for native Windows and Windows using WSL).
AppVeyor doesn't allow you to override single scripts from configuration to configuration, but only allows you to override whole script-stacks (like `test_script`, for example).

The `for`-array allows you to set build-stacks for exactly one configuration (using `only`) or all configurations but one (using `except`).

In this piece of code, for example, I set some script-stacks only for the `Native` configuration.
https://github.com/benmosher/eslint-plugin-import/blob/8118170d6fc6d626829ccbedba4be10bd3b97c55/appveyor.yml#L29-L73

But I don't think it's handy to create a configuration for an old node & babel-version which don't work anyway (as you can see in AppVeyor's history)

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -39,12 +39,17 @@ for:
     # install modules
     - ps: >-
         if ($env:nodejs_version -eq "4") {
-          npm install -g npm@3;
+          cmd /c npm install -g npm@3;
         }
         if ($env:nodejs_version -in @("8", "10", "12")) {
-          npm install -g npm@6.10.3;
+          cmd /c npm install -g npm@6.14.5;
         }
     - npm install
+
+    - ps: >-
+        if ([int]$env:nodejs_version -le 8) {
+          cmd /c npm i eslint@6 2`>`&1;
+        }
 
     # fix symlinks
     - git config core.symlinks true
@@ -53,6 +58,20 @@ for:
     # todo: learn how to do this for all .\resolvers\* on Windows
     - cd .\resolvers\webpack && npm install && cd ..\..
     - cd .\resolvers\node && npm install && cd ..\..
+
+    # Upgrade nyc
+    - npm i --no-save nyc@15.0.1
+    - ps: >-
+        $resolverDir = "./resolvers";
+        Get-ChildItem -Directory $resolverDir |
+        ForEach-Object {
+          Push-Location $(Resolve-Path $(Join-Path $resolverDir $_));
+          cmd /c npm ls nyc 2`>`&1;
+          if ($?) {
+            cmd /c npm i --no-save nyc@15.0.1 2`>`&1;
+          }
+          Pop-Location;
+        }
 
   # Post-install test scripts.
   test_script:

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "test-compiled": "npm run prepublish && BABEL_ENV=testCompiled mocha --compilers js:babel-register tests/src",
     "test-all": "node --require babel-register ./scripts/testAll",
     "prepublish": "not-in-publish || npm run build",
-    "coveralls": "nyc report --reporter lcovonly && cat ./coverage/lcov.info | coveralls"
+    "coveralls": "nyc report --reporter lcovonly && coveralls < ./coverage/lcov.info"
   },
   "repository": {
     "type": "git",

--- a/package.json
+++ b/package.json
@@ -81,6 +81,7 @@
     "glob": "^7.1.6",
     "in-publish": "^2.0.0",
     "linklocal": "^2.8.2",
+    "lodash.isarray": "^4.0.0",
     "mocha": "^3.5.3",
     "npm-which": "^3.0.1",
     "nyc": "^11.9.0",

